### DISCUSSION TITLE
GUI does not recognize super + action

### DISCUSSION THREAD

**Discussion by fuzzy7k** [AntiMicroX/antimicrox#748] [Q&A] [answered]
I am using antimicrox to control my GUI under linux. Hotkeys are the name of the game. Most of them work, like Alt+Tab. However, Super + key actions are ineffective. 

Using evtest, I find that antimicrox reports the following:
"Super_L" =  keycode 139 = KEY_MENU
"Menu" = keycode 127 = KEY_COMPOSE

On my real keyboard:
left super = keycode 125 = KEY_LEFTMETA
right super = keycode 126 = KEY_RIGHTMETA
menu = keycode 127 = KEY_COMPOSE

Would it be acceptable to fix Super_L in the code?

**Comment by fuzzy7k** [comment, marked as answer]
I created and associated issue and MR. Please review as I did not test it. It seemed a simple enough fix. I did verify that modifying the value in my saved profile from 6b(decimal 139) to 7d(decimal 125) did fix the issue. Otherwise, I am unable to set the value from within antimicrox.

**Reply by pktiuk** [reply]
PR merged, this should not be a problem anymore https://github.com/AntiMicroX/antimicrox/pull/750
